odim-controller/helmcharts/etcd-ha/templates/etcd-ha-statefulset.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apps/v1
```

Comment:
etcd-ha-statefulset.yaml contains manifest for just one instance, please check.

Reply:
Addressed